src/octopeer-github/main/Database/EventObject/EventFactory.ts

```diff
@@ -28,22 +28,42 @@ abstract class EventFactory {
 
     /**
      * Creates a KeystrokeEvent object.
-     * @param keystroke     the keystroke that is being logged.
-     * @param key_down_at   the timestamp when the key went down.
-     * @param key_up_at     the timestamp when the key went up.
-     * @returns {KeystrokeEvent} A KeystrokeEvent object that can be posted to the database.
+     * @param keystroke         the keystroke that is being logged.
+     * @param keystroke_type    the type of the keystroke (KEY_DOWN or KEY_UP)
+     * @param created_at        the timestamp when the keystroke was logged.
+     * @returns {EventObject} A KeystrokeEvent object that can be posted to the database.
      */
-    public static keystroke(keystroke: string, key_down_at: UnixTimestamp, key_up_at: UnixTimestamp): EventObject {
+    public static keystroke(keystroke: string, keystroke_type: KeystrokeType, created_at = EventFactory.getTime()): EventObject {
```

Comment:
One branch of the created_at hasn't been covered :innocent: 
By covering this one as well, EventObject will have 100% coverage in all coverage types. :)